backend/src/models/likes.js

```diff
@@ -25,9 +22,7 @@ class Likes {
         return this.db
             .query('SELECT value AS userLiked FROM likes WHERE objectType=? AND objectID=? AND userID=?',
                 [type, id, userID])
-            .then((userLiked) => {
-                return userLiked || { userLiked: 0 }
-            })
+            .then((userLiked) => ({ userLiked: userLiked || 0 }))
```

Comment:
This returns { userLiked: 0 } or { userLiked: { userLiked: 1 } }
It breaks comments, but not questions or reviews for some reason...